tests/functional/exporters/testing/test_http.py

```diff
@@ -1,121 +1,109 @@
-import os
 import json
 import threading
-import tempfile
 
+import pytest
 from six.moves.BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
 
-from testplan.testing.multitest import MultiTest, testsuite, testcase
-
+from testplan.testing import multitest
```

Comment:
This change looks fine to me.
Only one question why not directly import MultiTest, testsuite & testcase? of course it makes no difference, then in test_json.py and test_xml.py we 'd better have the same style. These test files are similar in structure.

Reply:
Importing testsuite/testcase at top levels was causing pytest to think that they were also pytest testcases (it just looks for any callable starting with "test"), it's not a big deal but was causing the output to be a bit confusing since pytest claimed to run more testcases than I expected it to. Importing the multitest module avoids that issue. I didn't bother to change all of the other imports, I agree consistency is better though.

In general, it is considered better practice to import whole modules rather than individual functions/classes/objects (e.g. the Google python style guide recommends it: http://google.github.io/styleguide/pyguide.html#22-imports)